client/src/Transaction.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+
+function Transaction(props: { nameOfCoin: string, timeStamp: string, purchasePrice:number, soldPrice: number}) {
+    // props: image, name, price
+
+    return (
+        <span id = "transaction">
```

Comment:
Not sure if we need spans here, it might just be worth it to wrap things in a div with inline: block or something similar